examples/example_plugin/example_plugin/forms.py

```diff
@@ -49,14 +49,22 @@ class Meta:
         nullable_fields = []
 
 
-class AnotherExampleModelForm(NautobotModelForm):
-    """Generic create/update form for `AnotherExampleModel` objects."""
+class AnotherExampleCreateModelForm(NautobotModelForm):
```

Comment:
```suggestion
class AnotherExampleModelCreateForm(NautobotModelForm):
```
This would be more intuitive perhaps? Since it is {Model_Name}CreateForm instead of splitting the `AnotherExampleModel` apart and inserting `Create` in the middle